components/src/molecules/tabs/Tabs.spec.js

```diff
@@ -0,0 +1,23 @@
+import { mount } from '@vue/test-utils'
+import ATabs from './Tabs.vue'
+
+describe('Tabs', () => {
+  it('has default structure', () => {
+    const wrapper = mount(ATabs)
+
+    expect(wrapper.is('div')).toBe(true)
+    expect(wrapper.classes()).toContain('a-tabs')
+    expect(wrapper.classes().length).toBe(1)
+  })
+
+  it('renders slot text when passed', () => {
```

Comment:
I'm not sure about this test, I think we should try to mount few `a-tab` and test some logic for changing tab by clicking on tab button. Please check if we can do something more to test it